src/Locator/ExternalCommand.php

```diff
@@ -36,8 +38,17 @@ public static function loadWithPaths(Paths $paths, ExecutableFinder $executableF
 
     public function locate(string $command): string
     {
-        // Search executable:
-        $executable = $this->executableFinder->find($command, null, [$this->binDir]);
+        $executable = false;
+        foreach ($this->suffixes as $suffix) {
+            $cmdName = $command . $suffix;
+            // Search executable:
+            $executable = $this->executableFinder->find($cmdName, null, [$this->binDir]);
+
+            if ($executable) {
```

Comment:
Instead of break, you could return it right away.
That makes the code less confusing and easier to read.
In that case, you can drop the executable assignment above the loop and throw the exception underneath the loop without the if statement.